go.mod

```diff
@@ -1,102 +1,101 @@
 module github.com/openshift-kni/cnf-features-deploy
 
-go 1.17
+go 1.18
 
 require (
 	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f
 	github.com/coreos/ignition v0.35.0
 	github.com/gatekeeper/gatekeeper-operator v0.2.1
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
 	github.com/golang/glog v1.0.0
-	github.com/google/go-cmp v0.5.8
+	github.com/google/go-cmp v0.5.9
 	github.com/ishidawataru/sctp v0.0.0-20210707070123-9a39160e9062
+	github.com/k8snetworkplumbingwg/multi-networkpolicy v0.0.0-20220908143610-19b7d2ba63f9
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.3.0
 	github.com/k8snetworkplumbingwg/sriov-network-operator v0.0.0-00010101000000-000000000000
-	github.com/kennygrant/sanitize v1.2.4
 	github.com/lack/mcmaker v0.0.6
 	github.com/lack/yamltrim v0.0.1
 	github.com/metallb/metallb-operator v0.0.0-00010101000000-000000000000
-	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.22.1
+	github.com/onsi/ginkgo/v2 v2.7.0
+	github.com/onsi/gomega v1.24.1
 	github.com/open-policy-agent/frameworks/constraint v0.0.0-20211123155909-217139c4a6bd
 	github.com/open-policy-agent/gatekeeper v0.0.0-20211201075931-d7de2a075a41
-	github.com/openshift-kni/numaresources-operator v0.0.0-00010101000000-000000000000
+	github.com/openshift-kni/k8sreporter v1.0.1
+	github.com/openshift-kni/numaresources-operator v0.4.12-0.2022111801.0.20230119155952-376b0302246a
 	github.com/openshift-psap/special-resource-operator v0.0.0-00010101000000-000000000000
 	github.com/openshift/api v3.9.0+incompatible
 	github.com/openshift/client-go v3.9.0+incompatible
 	github.com/openshift/cluster-nfd-operator v0.0.0-00010101000000-000000000000
 	github.com/openshift/cluster-node-tuning-operator v0.0.0-00010101000000-000000000000
-	github.com/openshift/machine-config-operator v0.0.1-0.20220706180257-35d79621a587
-	github.com/openshift/ptp-operator v0.0.0-20221109224215-0229788f84f5
-	github.com/spf13/cobra v1.4.0
-	github.com/stretchr/testify v1.7.1
-	golang.org/x/sys v0.0.0-20220722155257-8c9f86f7a55f
+	github.com/openshift/machine-config-operator v0.0.1-0.20230118083703-fc27a2bdaa85
+	github.com/openshift/ptp-operator v0.0.0-20230120025914-ae56875d0f51
```

Comment:
please put this one in the replace section below with all the other cnf related projects

Reply:
done